.travis.yml

```diff
@@ -13,7 +13,11 @@ install:
 script:
   - cd test
   - pip freeze  # so to help eventual debug: know what exact versions are in use can be rather useful.
-  - nosetests -xv --process-restartworker --processes=1 --process-timeout=999999999  --with-cov --cov=alignak
+  - which nosetests
+  - nosetests -V
```

Comment:
could be put in setup_test.sh but what's wrong with having it in travis ?  it's really with travis that I wanted to be sure of the versions used.. so my add in .travis file :)